app/helpers/react_components/mentoring/discussion.rb

```diff
@@ -9,6 +9,10 @@ def to_s
           {
             discussion_id: discussion.uuid,
             user_id: current_user.id,
+            mentor: {
+              handle: mentor.handle,
+              avatar_url: mentor.avatar_url
+            },
```

Comment:
I've added this new key for displaying the info of the mentor. You could add these as prefixed keys (e.g. `mentor_handle`) into the `mentor_solution` block instead if you prefered that. I don't have a huge preference.